applications/web/templates/deployment.yaml

```diff
@@ -406,11 +406,7 @@ spec:
           image: gcr.io/cloudsql-docker/gce-proxy:1.17
           command:
             - "/cloud_sql_proxy"
-            {{- if .Values.cloudsql.additionalConnection.enabled }}
-            - "-instances={{ .Values.cloudsql.connectionName }}=tcp:{{ .Values.cloudsql.dbPort }},{{ .Values.cloudsql.additionalConnection.connectionName }}=tcp:{{ .Values.cloudsql.additionalConnection.dbPort }}"
-            {{- else }}
-            - "-instances={{ .Values.cloudsql.connectionName }}=tcp:{{ .Values.cloudsql.dbPort }}"
-            {{- end }}
+            - "-instances={{- include "cloudsql.connectionString" . -}}"
```

Comment:
If there are no connections defined this argument will be empty. I haven't tested if this causes any issues, but I think we should add a condition or default to avoid this

Reply:
Hmm this can be done but reading the templating becomes unintuitive. I think if people enable cloud sql but provide no connections, it is fair for their applications to fail. It will still template but will error when they reach out to it.